src/api/stream_manager.py

```
@@ -0,0 +1,146 @@
+import os
+import time
+import threading
+import subprocess
+import cv2
+from pathlib import Path
```

Ruff (0.12.2)

`pathlib.Path` imported but unused

Remove unused import: `pathlib.Path`

(F401)